docker-compose-release.yaml

```diff
@@ -1,9 +1,19 @@
 version: '2.3'
 services:
   xl-release:
+    image: digitalai/continuum-dev-xlrelease
```

Comment:
Maybe this should just be "release-delivery-insights-dev"?  I don't know if there's a naming convention for Docker tags that aren't intended to ever be published.

Reply:
`release-delivery-insights-dev` sounds good to me